### PR TITLE
add mkdoc to docker image

### PR DESCRIPTION
**Pull request by nabuskey**
fixes: #36

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -89,6 +89,14 @@ RUN curl -L -O https://github.com/cnoe-io/cnoe-cli/releases/download/v0.1.0/cnoe
 COPY ./cnoe-wrapper.sh /usr/bin/cnoe
 RUN chmod +x /usr/bin/cnoe
 
+RUN apt-get update && apt-get install -y python3 python3-pip python3-venv
+
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+RUN pip3 install mkdocs-techdocs-core
```
Comment (omrishiv) [inline Dockerfile:98]: Can we add the versions to these so we don't get any unexpected surprises

File: Dockerfile [hunk 2]
```diff
@@ -89,6 +89,14 @@ RUN curl -L -O https://github.com/cnoe-io/cnoe-cli/releases/download/v0.1.0/cnoe
 COPY ./cnoe-wrapper.sh /usr/bin/cnoe
 RUN chmod +x /usr/bin/cnoe
 
+RUN apt-get update && apt-get install -y python3 python3-pip python3-venv
```
Comment (omrishiv) [inline Dockerfile:92]: We do a lot of apt work; can we merge these a bit? Also maybe delete the cache before we finish

**Review by omrishiv — CHANGES_REQUESTED** [review]
Thanks for taking care of this; just a few small nits

File: Dockerfile (hunk 1 above)
Comment (nabuskey) [reply Dockerfile]: Great idea. Done!

File: Dockerfile (hunk 2 above)
Comment (nabuskey) [reply Dockerfile]: I merged it with the kubectl install step above. Cache is mounted so the end image doesn't have any.

Comment (omrishiv) [reply Dockerfile]: Ah, I see that now, thanks!

**Review by omrishiv — APPROVED** [review]
Thanks for cleaning this up! LGTM

**Pull request merged**